src/vstart.sh

```diff
@@ -135,6 +136,7 @@ usage=$usage"\t--bluestore use bluestore as the osd objectstore backend\n"
 usage=$usage"\t--memstore use memstore as the osd objectstore backend\n"
 usage=$usage"\t--cache <pool>: enable cache tiering on pool\n"
 usage=$usage"\t--short: short object names only; necessary for ext4 dev\n"
+usage=$usage"\t--no_lockdep disable lockdep\n"
```

Comment:
minor: (just ignore if you don't agree) To be consistent with other option names ('--nodaemon)', I would prefer if it were '--nolockdep' instead.